v0/util/enum/message.js

```diff
@@ -0,0 +1,9 @@
+const Message = {
```

Comment:
Let's figure out a way to have error specific codes as well. Not important and can be skipped for now.